pyproject.toml

```diff
@@ -10,7 +10,7 @@ authors = [
 license = { file = "LICENSE" }
 readme = "README.md"
 dynamic = ["version", "description"]
-requires-python = ">=3.7"
+requires-python = ">=3.8"
```

Comment:
Python 3.8 is mighty old, EOL in 6 months. Don't you think we can make the jump to 3.10 and get some speed improvements?

Reply:
This needs to have the same Python version support as conda-build. Does that include 3.10+ only?